examples/image_similarity.ipynb

```diff
@@ -1118,7 +1118,7 @@
     "    columns = 6\n",
     "    for (i, image) in enumerate(images):\n",
     "        label_id = int(labels[i])\n",
-    "        ax = plt.subplot(len(images) / columns + 1, columns, i + 1)\n",
+    "        ax = plt.subplot(len(images) // columns + 1, columns, i + 1)\n",
```

Comment:
this is a platform-agnostic bug, if use `/` the output will be a float, and there will be ValueError "ValueError: Number of rows must be a positive integer, not 2.0", need integer division